migrate/scripts/watertower-tooltip.js

```diff
@@ -0,0 +1,17 @@
+const db  = require(../db.js)
+
+const ObjectID = require('mongodb').ObjectID
+const ID_String = "5d3f2b8672a1dcb5f8566562" //ID of corresponding entry.
+
+//Make the call directly to the database
+const createToolTip = id => {
+  return db.collection.insert(
```

Comment:
Shouldn't we be updating since the glyph already exists?

Reply:
This is a source of confusion for me. Before, the JSON for this entry looked like this: 
`{
    "_id" : ObjectId("5d3f2b8672a1dcb5f8566562"),
    "name" : "waterTowerIsland",
    "state" : {
        "graphic" : {},
        "style" : {
            "x" : "24.75%",
            "y" : "25%"
        }
    },
    "props" : {
        "size" : "8%",
        "hoverEffect" : "glow"
    },
    "animators" : {},
    "layer" : 4,
    "url" : "./images/waterTowerIsland.svg",
    "shape" : "svg"
}`

So the `"props"` did not have the attributes `props.tooltip` nor did it have `props.clickEffect`, so my assumption was that I'd need to use the `insert` function to _create_ these attributes. My thinking was that using `update` would require those to already exist, which wouldn't be the case here.

Reply:
Nope, `insert` creates a new element, but `update` can be used to add fields to an existing one.  Your logic makes a ton of sense, though.  `update` is saying that the element itself exists (the thing bounded by an ID).